glhe/topology/fluid_types.py

```diff
@@ -1,8 +1,8 @@
-from enum import Enum, auto
+from enum import Enum
```

Comment:
So I wasn't sure what enum library you are using.  Python comes with one native in both Python 2 and 3 now, but it doesn't have the auto() function.  I tried installing a third party, but I'm not sure if it was necessary, I dunno.  Anyway, I think if we are OK with avoiding the auto() function (which seems reasonable for the few values you have around here, then we can get away with only using the standard packaged enum library.  Your call obviously.  We could also just have an if check around the import where we fall back to our own "auto" implementation...but that seems a bit overkill.

Reply:
Manual numbering is fine. This is the documentation I was referencing: https://docs.python.org/3/library/enum.html#using-automatic-values

Reply:
Indeed a Python 3 thing.  I wouldn't mind using Python 3, I just for some reason couldn't get it to compile locally.  I think it's a problem with Python 3 and a bleeding edge version of GCC.  Also, what time is it there?

Reply:
If we can make it all work on 2.7 and 3.x, that would be good, if only for the reasons of me figuring out what needs to happen to make it work.

Time is 5:37 am in Malmö 😄It's hard to sleep when it looks like this outside.

![image](https://user-images.githubusercontent.com/2985979/40463285-bcc137e2-5f14-11e8-8458-b2508916e343.png)

I've been awake since 4:00 am anyway...

Reply:
Good luck getting back on normal routine time!

Reply:
Says the guy who's up at 1:19 am working on someone else's code 😄

Reply:
And maybe a little of my own...but yes yes I'm going to bed now...goodnight or morning